### DISCUSSION TITLE
wire:loading - image support?

### DISCUSSION THREAD

**Discussion by myriadcode** [livewire/livewire#985] [4. General]
Hi there,

I'm using `wire:loading` to show a placeholder. The data fetched contains images which are then embedded onto the page.

Whilst the placeholder shows during the initial fetching of data, while images get loaded it disappears. This creates an empty state where the `wire:loading` disappears and there is blank space until the images are loaded.

Using `wire:loading.class` does not work

**My question: Is it possible to keep wire:loading active until the elements fetched are loaded in the user's browser?**

**Comment by danharrin** [comment]
Hey, could you post the code that you tried with `wire:loading.class` please? Thank you.
